typescript/infra/config/environments/mainnet3/chains.ts

```diff
@@ -34,23 +31,6 @@ export const chainMetadataOverrides: ChainMap<Partial<ChainMetadata>> = {
       gasPrice: 1 * 10 ** 9, // 1 gwei
     },
   },
-  ethereum: {
-    blocks: {
-      confirmations: 3,
-    },
-    transactionOverrides: {
-      maxFeePerGas: 150 * 10 ** 9, // gwei
-      maxPriorityFeePerGas: 5 * 10 ** 9, // gwei
-    },
-  },
```

Comment:
removed overrides as discussed, also removing blocks.confirmations because registry has it set lower